tests/console/commands/test_add.py

```diff
@@ -1239,38 +1293,16 @@ def test_add_to_section_that_does_no_exist_yet(
 
     assert tester.io.fetch_output() == expected
 
+    assert isinstance(tester.command, InstallerCommand)
     assert tester.command.installer.executor.installations_count == 1
 
-    content = app.poetry.file.read()["tool"]["poetry"]
+    pyproject: dict[str, Any] = app.poetry.file.read()
+    content = pyproject["tool"]["poetry"]
 
     assert "cachy" in content["group"]["dev"]["dependencies"]
     assert content["group"]["dev"]["dependencies"]["cachy"] == "^0.2.0"
 
 
-def test_add_preferes_stable_releases(
```

Comment:
Why is this test removed?

Reply:
it's a copy of `test_add_prefers_stable_releases()`, but with a typo in the name